README.md

```diff
@@ -58,6 +58,30 @@ Install Nagios and setup the password for your nagiosadmin user.
       - user: nagiosadmin
         pass: S3cre7Passw0rd
 
+## Upgrading
+
+The role has automatic upgrade when you change the version with var: 
+
+    nagios_version: 4.3.2
+
+But the upgrade requires to remove some old files to, check [build-nagios.yml](tasks/build-nagios.yml) for details.
+
+## Supported versions
```

Comment:
Can you remove this "Supported versions" section and the info about tested with molecule? If there's a reason to move the testing framework to molecule that could perhaps be done in the future but I don't see any point to mentioning it here. Also the supported versions are already documented in https://github.com/networklore/ansible-role-nagios/blob/master/meta/main.yml. At this time it would make sense to remove Ubuntu 12.04 and add 16.04 instead. However I see this as another issue. 

Other than that I think this looks good now. You can leave the molecule.yml file but please remove the mention of it from README.md